### PR TITLE
document layouts (as "scale-aware transforms")

### PR DESCRIPTION
**Pull request by Fil**
covers dodge (#809), hexbin (#810)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1939,6 +1947,69 @@ This helper for constructing derived channels returns a [*channel*, *setChannel*
...
+### Dodge
+
+The dodge transform can be applied to any mark that consumes *x* or *y*, such as the Dot, Image, Text and Vector marks.
+#### Plot.dodgeY([*layoutOptions*, ]*options*)
+
+```js
+Plot.dodgeY({x: "date"})
+```
+
+If the marks are arranged along the *x* axis, the dodgeY transform piles them vertically, keeping their *x* position unchanged, and creating a *y* position that avoids overlapping.
```
Comment (ryoqun) [inline README.md:1963]: @Fil hi, well, i really wants this functionary. which topic branch is the most updated one (ideally based on plot v0.4.3)?

Comment (ryoqun) [reply README.md:1963]: specifically, i want these `Plot.link`s to be stacked (currently overlapping):
image

work-arounds like `opacity` and dashed lines isn't satisfactory... xD

and it seems i can specify the stacking direction with `anchor` with `top` & `bottom`? perfect. :)

Comment (Fil) [reply README.md:1963]: @ryoqun the current branch is https://github.com/observablehq/plot/tree/mbostock/reinitialize; please feel free to test it and give us feedback. (But note that it's highly probable that more changes will occur before this ships.)

**Comment by mbostock** [comment]
Merged into #823, which was merged into #801.

**Pull request closed without merging**